Providers/Modules/Plugins/Common/plugin/oms_configuration.rb

```diff
@@ -37,6 +38,11 @@ def test_onboard_file(file_name)
       end
 
       def get_proxy_config(proxy_conf_path)
+        old_proxy_conf_path = '/etc/opt/microsoft/omsagent/conf/proxy.conf'
```

Comment:
Since there are other files that must be changed in the DSC repo to account for the new/old proxy paths, this change should probably be excluded from your PR and included in the soon-to-come PR to change all of those files for multi-homing compatibility.

Feel free to effectively revert that commit in this file for your PR. This DSC resource update should only take in changes from your Diagnostics feature.